recipe/meta.yaml

```diff
@@ -21,10 +22,10 @@ requirements:
     - setuptools
 
   run:
-    - python >={{ python_min }}
-    - fair-research-login >=0.2.6,<0.3.0
-    - globus-sdk >=3.0.0,<4.0.0
-    - six
+    - python >={{ python_min }},<3.14
+    - sqlite
+    - six >=1.16.0
+    - globus-sdk >=3.15.0,<4.0.0
```

Comment:
I'm not sure if I can really speak to the other changes, but this part seems to match https://github.com/E3SM-Project/zstash/blob/main/conda/dev.yml.

```
  - sqlite
  - six >=1.16.0
  - globus-sdk >=3.15.0,<4.0
```

Reply:
@forsyth2 , it is an important job as a package developer to make sure these dependancy changes make sense, so we really need you to verify that.

It would be helpful if you could speak to at least the removal of fair-research-login as a dependency.